tests/test_e2e.py

```
@@ -1746,51 +1764,59 @@ def test_nested_list():
 
 
 def test_index():
-    res = query("UNWIND range(1, 100000) AS x CREATE (n:Node {v: x + 1})", write=True)
+    res = query("UNWIND range(1, 100000) AS x CREATE (n:Node {vi: x, vs: tostring(x)})", write=True)
     assert res.nodes_created == 100000
 
-    res = query("MATCH (n:Node {v: 5}) RETURN n.v")
-    assert res.result_set == [[5]]
+    memory_usage_before = memory_usage()
```

Ruff (0.12.2)

Local variable `memory_usage_before` is assigned to but never used

Remove assignment to unused variable `memory_usage_before`

(F841)